packages/opensrp-plans/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@opensrp/plans",
+  "version": "0.0.0",
+  "description": "Opensrp plans UI",
```

Comment:
What's the reason for specifically calling the "plans UI"?

Reply:
I think the opensrp plans package will hold the UI, functionality and ducks stuff so opensrp plans would suffice